metrics/cli/test/flutter/adapter/flutter_cli_service_adapter_test.dart

```diff
@@ -55,14 +56,48 @@ void main() {
     test(
       ".build() enables web support before building the web application",
       () async {
-        await flutterCliServiceAdapter.build(path);
+        await flutterAdapter.build(path);
 
         verifyInOrder([
           flutterCli.enableWeb(),
           flutterCli.buildWeb(path),
         ]);
       },
     );
+
+    test(
+      ".version() rethrows the StateError if showing the version throws it",
```

Comment:
```suggestion
      ".version() throws if showing the version throws",
```
Please, apply this suggestion for all similar situations.